### ISSUE TITLE
look for libc at runtime on windows

### ISSUE THREAD

**Issue by andrewrk** [zig-lang/zig#539]
moved from #302 

when using zig.exe outside of MSVC command prompt:

```
C:\Users\andy\tmp>zig.exe build-exe hello_libc.zig --library c
lld: error: could not open libcmtd.lib: no such file or directory
lld: error: could not open vcruntimed.lib: no such file or directory
```

this should just work if you have MSVC installed.

**Comment by andrewrk** [comment]
Workaround right now is to run zig.exe from the **VS2015 native tools command prompt**.

**Comment by andrewrk** [comment]
This is working but we should support VS2017 as well before closing the issue.

**Comment by andrewrk** [comment]
How rust solved the issue: https://github.com/rust-lang/rust/issues/38584

**Comment by andrewrk** [comment]
solved by @Dimenus in 38f05d4ac59aa799f947ae1fb4671acafcc283cb
